src/scss/vendor/_prism.scss

```diff
@@ -0,0 +1,143 @@
+/* Generated with http://k88hudson.github.io/syntax-highlighting-theme-generator/www */
```

Comment:
@bobwise  did you check on this about MIT?

Reply:
@yodasw16 signed off on this